container_pipeline/utils.py

```diff
@@ -1,10 +1,14 @@
+import hashlib
 import os
 
 
 def get_job_name(job_details):
     """Get jenkins job name from job_detials"""
-    return str(job_details['appid']) + "-" + str(job_details['jobid']) + "-" \
-        + str(job_details['desired_tag'])
+    namespace = str(job_details['appid']) + "-" + str(job_details['jobid']) + \
+        "-" + str(job_details['desired_tag'])
+    project = hashlib.sha224(namespace).hexdigest()
+
+    return project, namespace
```

Comment:
Because there are times when we only need `namespace` while times when we need `project` explicitly. the encoded value (project) is needed for only openshift project names, rest of the places, only namespace is required.